end my miserable life

PKPenguin321 opened the pull request:
https://www.youtube.com/watch?v=PTAXUYLbFYk

:cl: PKPenguin321
tweak: Sand now fits in your pockets
rscadd: Sand can now be thrown into people's eyes, doing slight stamina damage, making their vision slightly blurry, and making them stumble when they walk for a short time.
/:cl:

PKPenguin321 committed:
gives sand a new function
also makes it fit in your pocket

KazeEspada commented:
I did this years ago, but I kept bundling it with a screwdriver nerf. So it kept getting denied.

PKPenguin321 commented:
also this is untested and has no changelog as of yet so dont merge it too soon

Cheridan commented on code/modules/mining/ores_coins.dm:
```diff
@@ -60,6 +61,20 @@
+	var/mob/living/carbon/C = hit_atom
+	if((C.head && C.head.flags_cover & HEADCOVERSEYES) || \
+		(C.wear_mask && C.wear_mask.flags_cover & MASKCOVERSEYES))
+		visible_message("<span class='danger'>[C]'s mask/helmet blocks the sand!</span>")//glasses are not thick enough to stop it
```
>mask/helmet

lazy

Niknakflak replied:
>glasses are not thick enough to stop it

gimme a break

KorPhaeron replied:
That fucking visible message

PKPenguin321 replied:
Supposedly it'll pick the right one or some shit but like I already said this PR is not merge ready, I'm gonna touch this stuff up probably tomorrow. No bully

Jordie0608 replied:
You would need to insert the name of the mask or helmet as a [var] to get it to use the correct name; 'make/helmet' does nothing

PKPenguin321 committed:
splits check into 3 checks, changes sand effects
it no longer causes permanent eye damage. now it does slightly more eye blur than before, does 15 stamina damage (same as before), and causes some confusion.
also a check for glasses to block the sand

PKPenguin321 commented:
okay, i've added a changelog, fixed to the code so that it isn't absolute garbage, and actually tested it in-game. this is good for review/merge

optimumtact commented:
I give this a merge conflict/10

:+1:

PKPenguin321 commented:
ahaha i'm gonna kill myself

PKPenguin321 committed:
Merge branch 'master' of https://github.com/tgstation/-tg-station into wingo

# Conflicts:
#	code/modules/mining/ores_coins.dm

PKPenguin321 commented:
FUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUCKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKK

Xhuis commented:
![](http://www.reactiongifs.us/wp-content/uploads/2014/01/pocket_sand_king_of_the_hill.gif)
